packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -78,8 +78,10 @@ export const runSessionExpiryKey = "Fluid.GarbageCollection.RunSessionExpiry";
 export const trackGCStateKey = "Fluid.GarbageCollection.TrackGCState";
 // Feature gate key to turn GC sweep log off.
 export const disableSweepLogKey = "Fluid.GarbageCollection.DisableSweepLog";
-// Feature gate key to tombstone datastores.
-export const testTombstoneKey = "Fluid.GarbageCollection.Test.Tombstone";
+// Feature gate key to disable the tombstone feature, i.e., tombstone information is not read / written into summary.
+export const disableTombstoneKey = "Fluid.GarbageCollection.DisableTombstone";
+// Feature gate to enable throwing an error when tombstone object is used.
+export const enableThrowOnTombstoneUsageKey = "Fluid.GarbageCollection.ThrowOnTombstoneUsage";
```

Comment:
Along the same veins, `throwOnTombstoneKey`?

Reply:
Renamed.